composer.json

```diff
@@ -19,10 +19,10 @@
         "sign-in-with-apple"
     ],
     "require": {
-        "league/oauth2-client": "2.4.1",
+        "league/oauth2-client": "^2.0",
```

Comment:
Isn't it preferable to use an actual complete version? Unless we're absolutely sure that semver is followed properly, I've seen (even today) a `x.y.(z+1)` break an interface.